integr8ly/integreatly-operator
# MGDAPI-4377 Bump observability-operator to v3.0.13

# Issue link
JIRA: https://issues.redhat.com/browse/MGDAPI-4377

# What
This PR bumps the version of the observability-operator from v3.0.10 to v3.0.13.

# Verification steps

1. [Provision](https://qaprodauth.cloud.redhat.com/openshift/create) an OSD cluster.
2. Checkout the latest stable version of RHOAM: `git checkout tags/rhoam-v1.24.1-rc1 -b rhoam-v1.24.1-rc1`
3. Once the cluster is ready, target it and prepare it for a local install: `USE_CLUSTER_STORAGE=true INSTALLATION_TYPE=managed-api make cluster/prepare/local`
4. Install RHOAM locally: `INSTALLATION_TYPE=managed-api make code/run`
5. Once the installation is complete, stop running the operator locally and checkout this PR
6. Confirm that the installed version of the observability-operator is still v3.0.10: `oc get ClusterServiceVersion -n redhat-rhoam-observability-operator observability-operator.v3.0.10`
7. Run the operator locally again: `INSTALLATION_TYPE=managed-api make code/run`
8. The observability-operator will upgrade from v3.0.10 -> v3.0.11 -> v3.0.12 -> v3.0.13 
9. After a few minutes, verify that the installed version of the observability-operator is now v3.0.13: `oc get ClusterServiceVersion -n redhat-rhoam-observability-operator observability-operator.v3.0.13`
